spec/teaspoon_env.rb

```diff
@@ -1,7 +1,7 @@
 # Set RAILS_ROOT and load the environment if it's not already loaded.
 unless defined?(Rails)
-  ENV["RAILS_ROOT"] = File.expand_path("../../", __FILE__)
-  require File.expand_path("../../config/environment", __FILE__)
+  ENV["RAILS_ROOT"] = File.expand_path('..', __dir__)
+  require File.expand_path('../config/environment', __dir__)
 end
 
 Teaspoon.configure do |config|
```

Comment:
What in the heck is "teaspoon"? I just decided gem names have now officially jumped the shark 🦈

Reply:
Apparently "[Teaspoon is a Javascript test runner built for Rails.](https://github.com/jejacks0n/teaspoon)"